with_ssr/app/page.js

```diff
@@ -0,0 +1,43 @@
+import { SearchForm, SearchFormWrapper } from "@/src/entities";
+import { MovieListPage } from "@/src/pages/MovieListPage/MoviesListPage";
+
+export const getMoviesData = async (searchParams) => {
+  try {
+    const queryParams = new URLSearchParams({
+      sortBy: searchParams.sort,
+      search: searchParams.search,
+      filter: [searchParams.genre],
+    });
+    const response = await fetch(`${MAIN_API}?${queryParams}`);
+    const movies = await response.json();
+    return movies;
+  } catch (error) {
+    const movies = [
```

Comment:
mocked data should be moved to another file

Reply:
done